src/js/components/TaskDetailComponent.jsx

```diff
@@ -164,11 +167,17 @@ var TaskDetailComponent = React.createClass({
       );
     });
 
+    var ipAddressFields = this.getIpAddresses();
+    if (ipAddressFields != null && ipAddressFields.length > 0) {
+      ipAddressFields.unshift(<dt key="ip-addresses">IP Addresses</dt>)
+    }
```

Comment:
Nice one. What do you think about moving this logic inside `getIpAddresses`, though? I guess we can save an extra check?